docs/guides/references/_changelogs/12.4.0.md

```diff
@@ -0,0 +1,50 @@
+## 12.4.0
+
+_Released 1/24/2023_
+
+**Features:**
+
+- Added new
+  [`experimentalSkipDomainInjection`](/guides/references/experiments#Experimental-Skip-Domain-Injection)
+  flag to disable Cypress from setting `document.domain` on injection. Addresses
+  [#2367](https://github.com/cypress-io/cypress/issues/2367),
+  [#23958](https://github.com/cypress-io/cypress/issues/23958),
+  [#24290](https://github.com/cypress-io/cypress/issues/24290), and
+  [#24418](https://github.com/cypress-io/cypress/issues/24418).
+- Added new
+  [`experimentalMemoryManagement`](/guides/references/experiments#Configuration)
+  flag to improve memory management in Chromium-based browsers. Addresses
+  [#23391](https://github.com/cypress-io/cypress/issues/23391).
+- [`.as`](/api/commands/as) now accepts an options argument, allowing an alias
+  to be stored as a "query" or "static" value. Addresses
+  [#25173](https://github.com/cypress-io/cypress/issues/25173).
+- Official support for Vite 4 projects for component testing. Addresses
```

Comment:
```suggestion
- Added official support for Vite 4 projects for component testing. Addresses
```

Just to be consistent.

Reply:
I actually think this should be top line of the changelog.  Ahead of experimental flags.